qa/rpc-tests/mergetoaddress_helper.py

```diff
@@ -358,8 +358,4 @@ def run_test(self, test):
         assert_equal(result["mergingNotes"], Decimal('2'))
         assert_equal(result["remainingNotes"], num_notes - 4)
         wait_and_assert_operationid_status(test.nodes[0], result['opid'])
-        # Don't sync node 2 which rejects the tx due to its mempooltxinputlimit
-        sync_blocks(test.nodes[:2])
-        sync_mempools(test.nodes[:2])
-        test.nodes[1].generate(1)
```

Comment:
Removing this `generate` line might adversely affect the test. Instead, replace the `sync_blocks` and `sync_mempools` lines with a `test.sync_all()`.

Reply:
It looked like it was just making sure that the mempools could sync after removing the `mempooltxinputlimit` rejecting transaction (remove by mining it). But I guess it could be checking that all nodes accept these txsx in a block. Will re-add.